ext/gd/gd.c

```diff
@@ -57,13 +57,13 @@
 
 #include "gd_compat.h"
 
-#include <gd.h>
-#include <gd_errors.h>
-#include <gdfontt.h>  /* 1 Tiny font */
-#include <gdfonts.h>  /* 2 Small font */
-#include <gdfontmb.h> /* 3 Medium bold font */
-#include <gdfontl.h>  /* 4 Large font */
-#include <gdfontg.h>  /* 5 Giant font */
+#include "libgd/gd.h"
+#include "libgd/gd_errors.h"
+#include "libgd/gdfontt.h"  /* 1 Tiny font */
+#include "libgd/gdfonts.h"  /* 2 Small font */
+#include "libgd/gdfontmb.h" /* 3 Medium bold font */
+#include "libgd/gdfontl.h"  /* 4 Large font */
+#include "libgd/gdfontg.h"  /* 5 Giant font */
```

Comment:
When building against system libgd, this would include the headers of the bundled libgd. I think we need to have separate logic for system vs. external, i.e.
````.c
#ifdef HAVE_GD_BUNDLED
# include "libgd/gd.h"
…
#else
# include <gd.h>
…
#endif
````

Reply:
That sounds right @cmb69, I'll change according to the suggestion you made :-)